test/E2ETest/Microsoft.Test.E2E.AspNet.OData/ModelBuilder/RecursiveComplexTypesE2ETests.cs

```diff
@@ -143,11 +143,11 @@ private static IEdmModel GetEdmModel(WebRouteConfiguration configuration)
             ""Address"": {
                 ""Street"": ""123 Main Street"",
                 ""City"": ""Seattle"",
-                ""Country"": ""USA"",
```

Comment:
I don't think "USA" or "Canada" are policheck violations. Maybe we should change the "Country"?

Reply:
It's open source. Don't use the country name in our source/test codes. if you find more, please correct them.

Reply:
Excerpt from the policheck details:
--------
<Term Term="countries" Severity="2" TermClass="Geopolitical" CaseSensitive="" WholeWord="" CommentSearch="" RowCount="1" TermVersionId="5" FieldsImpacted="Guidelines" TermId="79569"><TermTranslation>English Translation</TermTranslation><Comment>For detailed info see - http://policheck.azurewebsites.net/Pages/TermInfo.aspx?LCID=9&amp;TermID=79569</Comment><Actions>For detailed info see - http://policheck.azurewebsites.net/Pages/TermInfo.aspx?LCID=9&amp;TermID=79569<Context>When used in a legal, international standards or similar context, such as in license terms (EULAs).</Context><ActionRecommendation>&lt;b&gt;Leave term unchanged&lt;/b&gt;</ActionRecommendation><Context>When used in any other context.</Context><ActionRecommendation>Ask your Microsoft contact for guidance.</ActionRecommendation><Context>When used to label a country/region list or similar resource.</Context><ActionRecommendation>&lt;b&gt;REPLACE WITH&lt;/b&gt; "country/region"</ActionRecommendation></Actions></Term>
-------

Reply:
That's the content in the file I sent to you. So, What's the value to copy/paste here?

Reply:
The excerpt provides specific information about what is causing the error, and the proposed fix: 
`When used to label a country/region list or similar resource.<b>REPLACE WITH</b> "country/region"`

Reply:
I changed this.